### PR TITLE
eslint: add German translation

### PR DESCRIPTION
**Pull request by micschwarz**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/eslint.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# eslint
+
+> Ein erweiterbarer Linter für JavaScript und JSX.
+> Mehr Informationen: <https://eslint.org>.
```
Comment (kbdharun) [inline pages.de/common/eslint.md:4]:
```suggestion
> Weitere Informationen: <https://eslint.org>.
```

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution.

**Translate all command arguments inside `{{}}`.**

File: pages.de/common/eslint.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# eslint
+
+> Ein erweiterbarer Linter für JavaScript und JSX.
+> Weitere Informationen: <https://eslint.org>.
+
+- Erstellen einer ESLint Konfigurationsdatei:
```
Comment (pixelcmtd) [inline pages.de/common/eslint.md:6]:
```suggestion
- Erstelle eine ESLint-Konfigurationsdatei:
```

File: pages.de/common/eslint.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`eslint --init`
+
+- Linter auf mehreren Dateien ausführen:
```
Comment (pixelcmtd) [inline pages.de/common/eslint.md:10]:
```suggestion
- Linte Dateien:
```

File: pages.de/common/eslint.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`eslint {{dateiname}}.js {{dateiname1}}.js`
+
+- Linterfehler beheben:
```
Comment (pixelcmtd) [inline pages.de/common/eslint.md:14]:
```suggestion
- Behebe Lintingfehler:
```

File: pages.de/common/eslint.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`eslint --fix`
+
+- Linter mit einer Konfigurationsdatei ausführen:
```
Comment (pixelcmtd) [inline pages.de/common/eslint.md:18]:
```suggestion
- Linte mit einer Konfigurationsdatei:
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
What I said in #9406, also applies here.

File: pages.de/common/eslint.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+> Ein erweiterbarer Linter für JavaScript und JSX.
+> Weitere Informationen: <https://eslint.org>.
+
+- Erstellen einer ESLint Konfigurationsdatei:
+
+`eslint --init`
+
+- Linter auf mehreren Dateien ausführen:
+
+`eslint {{dateiname}}.js {{dateiname1}}.js`
```
Comment (EmilySeville7cfg) [inline pages.de/common/eslint.md:12]:
```suggestion
`eslint {{path/to/file1.js path/to/file2.js ...}}`
```

Mandatory fix (requires the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/eslint.md) page to be changed): [wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths) path format is used, `eslint` doesn't require exactly two files to be passed.
Note: translation is based on google translate.

File: pages.de/common/eslint.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Linterfehler beheben:
+
+`eslint --fix`
+
+- Linter mit einer Konfigurationsdatei ausführen:
+
+`eslint -c {{pfad/zu/konfigurationsdatei}} {{app/src}}`
```
Comment (EmilySeville7cfg) [inline pages.de/common/eslint.md:20]:
```suggestion
`eslint -c {{pfad/zu/konfigurationsdatei}} {{path/to/app_source}}`
```

Mandatory fix (requires the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/eslint.md) page to be changed): [wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths) path format is used.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Changes requested because errors found.
- This page should be reviewed by `eslint` users.
- Thanks for a contribution! 🚀

File: pages.de/common/eslint.md (hunk 6 above)
Comment (pixelcmtd) [reply pages.de/common/eslint.md:12]: wdym translation is based on google translate? you literally didn't translate it. also if you disagree with something in a translation, please open a separate pr fixing it in the original page

Comment (EmilySeville7cfg) [reply pages.de/common/eslint.md:12]: Hm... Yes, my mistake. Thanks!

File: pages.de/common/eslint.md (hunk 7 above)
Comment (EmilySeville7cfg) [reply pages.de/common/eslint.md:20]: @pixelcmtd, without `_` I get this translation: `Pfad/zu/App-Quelle`. Should I change my suggestion?

File: pages.de/common/eslint.md (hunk 6 above)
Comment (kbdharun) [reply pages.de/common/eslint.md:12]: I think `P`  and `D` mustn't be capital letters.

```suggestion
`eslint {{pfad/zu/datei1.js pfad/zu/datei2.js ...}}`
```

File: pages.de/common/eslint.md (hunk 7 above)
Comment (pixelcmtd) [reply pages.de/common/eslint.md:20]: I think I'd use `pfad/zu/quellordner` and `path/to/source_directory`, how about that?

Comment (EmilySeville7cfg) [reply pages.de/common/eslint.md:20]: I've changed my suggestion to reflect your suggestion. :)

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**